node/src/chain_spec.rs

```diff
@@ -15,6 +15,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+#![allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
Now that `Eq` is derived for `Extensions`, do we still need to silence clippy for this warning?

Reply:
With our current master it is still needed.